middleware/logger.go

```diff
@@ -19,48 +12,26 @@ import (
 // print in color, otherwise it will print in black and white.
 //
 // Logger prints a request ID if one is provided.
-func Logger(next http.Handler) http.Handler {
-	fn := func(w http.ResponseWriter, r *http.Request) {
-		reqID := GetReqID(r.Context())
-		prefix := requestPrefix(reqID, r)
-		lw := wrapWriter(w)
-
-		t1 := time.Now()
-		defer func() {
-			t2 := time.Now()
-			printRequest(prefix, reqID, lw, t2.Sub(t1))
-		}()
+var Logger = Metrics(logMetricsHandler{})
```

Comment:
@VojtechVitek : I'm pretty sure that this line keep the old interface the same for a stable API.

You still can use the Logger as a function and maintain compatibility with code such as:

```go
r.Use(middleware.Logger)
```

_(https://github.com/pressly/chi/pull/56#discussion_r68762567)_